src/amber/router/route.cr

```diff
@@ -35,5 +35,20 @@ module Amber
     def call(context)
       handler.call(context)
     end
+
+    def parse_params(params)
```

Comment:
I liked `substitute_keys_in_path`, thanks I really couldn't think of a better name at the time.